utilities/calculateFinalPot.test.ts

```diff
@@ -0,0 +1,802 @@
+import calculateFinalPot from "./calculateFinalPot";
+import { GameType } from "../models/GameSituation";
+import { Rank, Suit } from "../models/PlayingCard";
+
+describe("calculateFinalPot(gameSituation)", () => {
+  describe("return the final pot size", () => {
+    describe("when the game ended at preflop", () => {
+      test("players have only one action in the street", () => {
+        expect(
+          calculateFinalPot({
+            type: GameType.cash,
+            playerLength: 2,
+            playerStackSizes: [100, 120],
+            playerCards: [null, null],
+            heroIndex: 0,
+            smallBlindSize: 0.5,
+            antiSize: 0,
+            preflop: {
+              actions: [
+                { playerIndex: 0, betSize: 3 },
+                { playerIndex: 1, betSize: 1 }
+              ]
+            }
+          })
+        ).toBe(4);
+      });
+
+      test("one player has one action and other have two in the street", () => {
+        expect(
+          calculateFinalPot({
+            type: GameType.cash,
+            playerLength: 3,
+            playerStackSizes: [100, 120],
+            playerCards: [null, null],
+            heroIndex: 0,
+            smallBlindSize: 0.5,
+            antiSize: 0,
+            preflop: {
+              actions: [
+                { playerIndex: 2, betSize: 2 },
+                { playerIndex: 0, betSize: 2 },
+                { playerIndex: 1, betSize: 4 },
+                { playerIndex: 2, betSize: 0 },
+                { playerIndex: 0, betSize: 0 },
+                { playerIndex: 1, betSize: 0 }
```

Comment:
We are using made-bet-size for player actions, right? On this premise, these actions look incorrect. Because player 2 raised at `actions[0]`, so player 2's bet size at `actions[3]` should be 2. For also player 0 and 1 are supposed to have bet size that 2 and 4.